src/GitHub.VisualStudio/Commands/GoToSolutionOrPullRequestFileCommand.cs

```diff
@@ -0,0 +1,228 @@
+﻿using System;
+using System.ComponentModel.Composition;
+using GitHub.Services;
+using GitHub.Extensions;
+using GitHub.VisualStudio;
+using GitHub.Services.Vssdk.Commands;
+using Microsoft.VisualStudio;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Editor;
+using Microsoft.VisualStudio.Shell.Interop;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.Text.Differencing;
+using Task = System.Threading.Tasks.Task;
+
+namespace GitHub.Commands
+{
+    [Export(typeof(IGoToSolutionOrPullRequestFileCommand))]
+    public class GoToSolutionOrPullRequestFileCommand : VsCommand, IGoToSolutionOrPullRequestFileCommand
```

Comment:
Could we have a comment here (and on the interface) as it's not totally clear from the class name what this command will do.

Reply:
Done. I hope it make sense. Let me know if it isn't clear.